pii_recognition/data_readers/data_test.py

```diff
@@ -0,0 +1,61 @@
+from typing import List
+
+import pytest
+
+from .data import Data
+from pii_recognition.labels.schema import Entity
+
+
+def test_Data_token_labels():
```

Comment:
these 2 tests seem to test `dataclass` functionality? I don't think they are useful for this project.

Reply:
That makes sense. Will remove the two tests.